config/base/params-vllm-gaudi.env

```diff
@@ -1 +1 @@
-vllm-gaudi-image=quay.io/opendatahub/vllm:fast-gaudi
```

Comment:
@trujillm may you rebase your PR on `incubation`?
Looks like you did your changes on top of the `main` branch, which is no longer our default branch, and there are a lot of changes because of the different base branch.

Reply:
@israel-hdez I believe I have corrected please verify and let me know if you need anything else from my side